### DISCUSSION TITLE
DKB CSV: ambiguous account name with shared accounts

### DISCUSSION THREAD

**Discussion by mrwsl** [firefly-iii/firefly-iii#8660] [Firefly III] [answered]
I'm having troubles importing the new DKB CSV for shared and personal accounts with the filter from [here](https://github.com/firefly-iii/import-configurations/tree/main/de/dkb). The new CSV format is lacking information, as in: The `Zahlungspflichtige*r` (`account-name`) might be ambiguous. So my name is sometimes used for the shared account and for my personal account. So the shared account might have the name `Forename Surname`, `FORENAME SURNAME` etc. as well as `Forename and Wifes' Forename Surname`. This way, importing data is quite cumbersome. Do you have a solution by any chance? 

I'm going to ask DKB to add `Opposing account (IBAN)` which is currently missing. This should resolve this problem as well, shouldn't it?

**Comment by mrwsl** [comment, marked as answer]
Update: I talked to DKB support, and they know the current CSV is mediocre compared to the old online banking. So for now, I just stick to the export from the old online banking.

**Reply by JC5** [reply]
That at least sounds positive, perhaps they will be quick to push updates.
